dist/ui.js

```diff
@@ -34,7 +34,7 @@ var UI = function (_EventEmitter) {
   function UI() {
     _classCallCheck(this, UI);
 
-    var _this = _possibleConstructorReturn(this, Object.getPrototypeOf(UI).call(this));
+    var _this = _possibleConstructorReturn(this, (UI.__proto__ || Object.getPrototypeOf(UI)).call(this));
```

Comment:
`__proto__` is deprecated, and use is highly discouraged.  With the way the command object is currently passed around, this could be destructive for downstream uses.

We should be using  [`UI.prototype.constructor`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/constructor) instead.